internal/integration/hclsqlspec/hclsqlspec_test.go

```diff
@@ -328,8 +328,9 @@ func TestMarshalTopLevel(t *testing.T) {
 	h, err := hcl.MarshalSpec(c)
 	require.NoError(t, err)
 	require.EqualValues(t, `column "column" {
-  null = true
-  type = varchar(255)
+  null    = true
+  type    = varchar(255)
+  comment = ""
```

Comment:
Why the `comment` is not omitted in this case?

Reply:
Because I want to add the `Marshal` in a second PR. This is needed to make the test pass.